.travis.yml

```diff
@@ -13,11 +13,15 @@ addons:
         repo_token:
           secure: "db3d67198bc95a577c6ba82351ae6c4167a45b070acd20da99cc9373c97331e1"
 before_script:
+  - psql -c 'create database food;' -U postgres
+  - psql -c "CREATE USER young WITH PASSWORD 'andela';" -U postgres
   - travis_wait npm install
   - npm install codeclimate-test-reporter -g
 script:
   - ./node_modules/mocha/bin/mocha --recursive './test/' --compilers js:babel-core/register --exit
+  - psql -c 'drop database food;' -U postgres
 after_success:
+  - psql -c 'create database food;' -U postgres
```

Comment:
Are you trying to drop this database? Not sure why you're creating after success. Please explain what this does.

Reply:
in Travis after running the test, for me to run the cover which has to run another test, I have to recreate the schemas in order not to  duplicate the entry in the tables thereby raising error